### ISSUE TITLE
Feature Request: Make Input display as Inline instead of block

### ISSUE THREAD

**Issue by mrin9** [AT-UI/at-ui#100]
Making ```<at-input>``` as close to HTML's ```<input>``` is the goal here. currently ```<at-input>```  occupy 100% width and display as a block element, making it difficult to place labels, icons etc on the sides.

It would be really great if you make the default behavior of  ```<at-input>```  as close to ```<input>``` if there are no technical challenges.

This might be a breaking change, so if not in this version please consider in a later version

**Comment by koppthe** [comment]
You are right. Make Input display as inline-block is better. Thanks for your suggestion.
